kvirt/config.py

```diff
@@ -300,7 +300,8 @@ def __init__(self, client=None, debug=False, quiet=False, region=None, zone=None
         self.overrides.update(config_data)
 
     def create_vm(self, name, profile, overrides={}, customprofile={}, k=None,
-                  plan='kvirt', basedir='.', client=None, onfly=None, wait=False, onlyassets=False):
+                  plan='kvirt', basedir='.', client=None, onfly=None, wait=False,
+                  onlyassets=False, cluster=None):
```

Comment:
why introduce this variable? 
it seems to be solely for the purpose of changing k3s workflow, in which case you can instead rely on the fact that overrides dict will contain a kube variable (pretty much your cluster variable) and a kubetype (k3s, generic or openshift)